extensions/wikia/AchievementsII/AchConfig.class.php

```diff
@@ -158,21 +158,21 @@ private function reset() {
 	 * @return AchConfig instance
 	 */
 	static public function getInstance() {
-		if (!isset(self::$mInstance))
+		if( !isset( self::$mInstance ) )
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add curly braces around the nested statement(s). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS121)